src/main/java/frc/team4909/robot/Robot.java

```diff
@@ -150,8 +150,8 @@ public void robotInit() {
     manipulatorGamepad.buttonHeld(BionicF310.RB, new HatchPanelIntakeOpen());
 
     /* Climber */
-    climberGamepad.buttonHeld(BionicF310.RB, new StiltsUpOnly());
-    climberGamepad.buttonHeld(BionicF310.LB, new StiltsDownOnly());
+    // climberGamepad.buttonHeld(BionicF310.RB, new StiltsUpOnly());
```

Comment:
why the change? has this implemented elsewhere ?

Reply:
Stilts up and down only is controlled through the RIght Joystick of the climber gamepad.